packages/cli/README.md

```diff
@@ -1,32 +1,31 @@
- - `--proxy`           Start proxy service  [default: true]          
- - `--nat`             NAT Traversal options for proxy [choices: "localhost", "lan", "extip"] 
--  `--persistentPort`  run the proxy on a persistent UDP port             
--  `--rpc`             Enable the JSON-RPC server with HTTP endpoint [default: true]
--  `--rpcPort`         HTTP-RPC server listening port             
--  `--rpcAddr`         HTTP-RPC server listening interface address [default: "localhost"]
--  `--metrics`         Turn on Prometheus metrics reporting [default: false]
--  `--metricsPort`     Port exposed for metrics scraping  [default: 18545]
--  `--dataDir`         data directory where content is stored        
-### Starting with the same Node ID 
+  --pk                base64 string encoded protobuf serialized private key  [string]
+  --bootnode          ENR of Bootnode                                        [string]
+  --bindAddress       initial IP address and UDP port to bind to             [string]
+  --bootnodeList      path to a file containing a list of bootnode ENRs      [string]
+  --rpc               Enable the JSON-RPC server with HTTP endpoint          [boolean] [default: true]
+  --rpcPort           HTTP-RPC server listening port                         [number] [default: 8545]
+  --rpcAddr           HTTP-RPC server listening interface address            [string]
+  --metrics           Turn on Prometheus metrics reporting                   [boolean] [default: false]
+  --metricsPort       Port exposed for metrics scraping                      [number] [default: 18545]
+  --dataDir           data directory where content is stored                 [string]
+  --web3              web3 JSON RPC HTTP endpoint for local Ethereum node    [string]
+  --networks          subnetworks to enable  (options are: `history`, `state`, `beacon`) [default: `history`]
+  --trustedBlockRoot  a trusted blockroot to start light client syncing of the beacon chain [string]
 
-To start a node that has the same node ID each time, you can pass the `--pk` parameter at start-up with a base64 string encoded protobuf serialized private key.  So `ts-node src/index.ts --pk=CAISINx/bjWlmCXTClX2JvDYehb8FSrE6l4MA9LGvP74XdfD` will always start the `cli` client with the node ID `2a9511ca767b7b56bb873234209557d07c5fe09382ed060b272c6a933c5658f5`.
+### Starting with the same Node ID 
 
-You can use the `generateKeys` script to generate any number of private keys using the command `ts-node scripts/generateKeys --numKeys=X` where X is the number of keys to generate.
+To start a node that has the same node ID each time, you can pass the `--pk` parameter at start-up with a base64 string encoded protobuf serialized private key.  So `ts-node-esm src/index.ts --pk=CAISINx/bjWlmCXTClX2JvDYehb8FSrE6l4MA9LGvP74XdfD` will always start the `cli` client with the node ID `2a9511ca767b7b56bb873234209557d07c5fe09382ed060b272c6a933c5658f5`.
 
 ### Connecting to the Portal Network
```

Comment:
I think you should really go away from the "to the Portal Network" semantics and move over to always reference a specific network.

Reply:
(or maybe it's also my lack of the understanding of the PN structure. So are bootnodes e.g. for all the networks together? Or are they always an entrypoint to all the different networks?

(if things like this are e.g. the case then I get the whole `to the Portal Network` notion better, then this might also help though to make correct relation here a bit clearer in the docs))

Reply:
At present, the bootnodes all just support the History network (and I can't connect to most of them anyway for whatever reason).  That's a big milestone goal for 2024 is to get a more functional "testnet" up and running with responsive nodes that can really handle some load.  The Ultralight bootnodes are all entirely hopeless and basically stop responding within an hour or two of a restart.